### ISSUE TITLE
Preparation for the Redesign

### ISSUE THREAD

**Issue by yatil** [w3c/wai-policies-prototype#314]
👋 The redesign is coming to this repository soon. (Hooray! 🎉) We need to check off several steps before we can link your content to the [redesign repository](https://github.com/w3c/wai-website). I marked some of the steps with RM meaning Resource Manager. Others are marked with @yatil, and I’ll do/help you through the technical parts.

In addition to this issue, you might find other issues in your repository that are labeled with wai-redesign-before that should be addressed/resolved before the redesign. That might mean to postpone working on it until after the redesign (use the label wai-redesign-after for those issues).

* [ ] **Content** is updated and ready to be put into the new design. (RM)
  * The best case scenario would be that the content is ready before doing step three below. However, that might not be feasible in every case, so @yatil might do the next step before the content is ready and just port the content that is available to him.
  * Once done, add label wai-redesign-content-done.
* [ ] **Tech** Create branch/repository with the new structure (@yatil)
  * For simple resources (separate pages/pages that don’t need a left navigation), you will find the documents in the root directory of the repository.
  * For more complicated resources, @yatil creates a Jekyll collection which adds the left sidebar navigation amongst other things. You will then find your files in a folder called `_resource_name`. 
   * Documentation should be available in the [w3c/wai-website wiki](https://github.com/w3c/wai-website/wiki/Repository-Anatomy-for-individual-resources) soon.
  * Once done, add label wai-redesign-repo-done.
* [ ] **Content** Use current content and port it to the new design (@yatil)
  * Once this is done, all previews will be in the new design.
  * Once done, add label wai-redesign-content-ported.
* [ ] **Content** Check if content renders well, make changes as needed/wanted. ([documentation](https://github.com/w3c/wai-website/wiki/Preparing-Content)) (RM)
  * Once done, add label wai-redesign-content-checked.
* [ ] **Tech** Link this repository to the w3c/wai-website repository (@yatil)
  * Once done, add label wai-redesign-repo-linked.
* When all checkpoints above are checked, close this issue.

**Comment by yatil** [comment]
Note that I will port the stuff that I did on the w3c/wai-website prototype to this. I don’t think there have been content changes in-between.

**Comment by iamjolly** [comment]
That is correct. Here have been no content.m changes, @yatil. Some changes will be made based on the usability testing, but best to make those in the repo after the new design is applied. Thank you!

**Comment by yatil** [comment]
@iamjolly  I have now ported the policies prototype to the new design in a separate branch: https://github.com/w3c/wai-policies-prototype/tree/neo

I also created a [release branch](https://github.com/w3c/wai-policies-prototype/tree/release) where we merge changes from the neo branch that should be used in w3c/wai-website.

Currently the GitHub page is still rendering the old version (until we figured out #313), so you have to build it locally. The following should work: 

~~~bash
$ bundle update
…
$ bundle exec jekyll serve
~~~

(You only need bundle update once after you checked it out, I think.)
